frontend/src/components/GShootEditor.vue

```diff
@@ -126,6 +125,29 @@ SPDX-License-Identifier: Apache-2.0
             <span>{{ showManagedFields ? 'Hide' : 'Show' }} managed fields</span>
           </v-tooltip>
         </div>
+        <div class="px-2">
+          <v-tooltip location="top">
+            <template #activator="slotProps">
+              <div v-bind="slotProps.props">
+                <v-btn
+                  variant="text"
+                  size="x-small"
+                  flat
+                  icon
+                  @click="renderWhitespaces = !renderWhitespaces"
+                >
+                  <!-- eslint-disable vue/no-v-html -->
+                  <span
+                    style="width: 18px; height: 18px"
+                    v-html="renderWhitespaces ? whitespaceEye : whitespaceEyeOff"
```

Comment:
```suggestion
                    v-html="renderWhitespaces ? whitespaceEyeOff : whitespaceEye"
```

Reply:
Can we discuss this? I agree as of now it is different than we handle the icon for the managed fields. But my gut feeling is that it would be wrong to show the off icon when we render the whitespaces. How do other tools handle this kind of flag buttons? I need to check this